templates/tftmpl/root.go

```diff
@@ -41,11 +41,15 @@ const (
 	// variable is written to.
 	TFVarsFilename = "terraform.tfvars"
 
+	// VarsTFVarsFileName is the file name for a tfvars file which is generated and contains
+	// variables provided as part of the task configuration
+	VarsTFVarsFileName = "variables.auto.tfvars"
```

Comment:
If `auto.tfvars` is used as the filename format now, we may not need to even track `varFile` anymore and pass that through to the TF CLI command since it should automatically discover it in the working directory.

Reply:
Oh, interesting, I will give this a try